### DISCUSSION TITLE
如何去掉表格标题下方和图表标题上方的空行？

### DISCUSSION THREAD

**Discussion by linelee02** [sjtug/SJTUThesis#896] [提问]
如标题：
![image](https://user-images.githubusercontent.com/72928012/221353885-c0dcfcc1-407a-488b-9d35-b3e065f9dca0.png)
![image](https://user-images.githubusercontent.com/72928012/221353907-6a6b8f2d-ea8c-470d-96c8-0f71de9832e6.png)

**Comment by LogCreative** [comment]
导言区添加
```latex
\captionsetup{skip=0pt}
```
以移除 `caption` 宏包设定的垂直间距。

**Reply by linelee02** [reply]
谢谢管理员
另外，如何将中英文标题之间的垂直间距也去掉呢？
![image](https://user-images.githubusercontent.com/72928012/221357602-55335ddc-c385-46a2-a4e2-d9aba3358e4c.png)

**Reply by LogCreative** [reply]
一种方法：
```latex
\DeclareBiCaptionSeparator{-1ex}{\vspace{-1ex}}
\captionsetup{bi-separator=-1ex}
```
以强制减少 `bicaption` 宏包设定的垂直间距。

**Reply by linelee02** [reply]
感谢万能的管理员，完美搞定。
